### ISSUE TITLE
In the generated docs, show `Optional: true/false` (or similar), when neither `!` nor `?` is specified in a prop

### ISSUE THREAD

**Issue by Kiarokh** [jgroth/kompendium#120]
If we have neither `!` nor `?` in a prop, the docs show this:

![image](https://user-images.githubusercontent.com/154725/224955556-be33b72b-1402-4b65-8f23-7ba5e36229a1.png)

**Comment by adrianschmidt** [comment]
Yeah, Johan and I have talked about this at some point. I was pretty convinced there was an issue already, but apparently not. Thanks for making one!

**Comment by adrianschmidt** [comment]
Yes. Let's only show `Required: true` if there's an `!`. Otherwise, we show `Required: false`. And let's never show "Optional".
